.travis.yml

```diff
@@ -10,5 +10,7 @@ cache:
 install: true
 script:
   - yarn
-  - yarn test
+  - yarn test -- --coverage
```

Comment:
Should already default to collecting coverage via the jest config, however an argument can be made here for explicitness.